plugin/cmake4vim.vim

```diff
@@ -43,12 +43,13 @@ let g:cmake_build_executor = get(g:, 'cmake_build_executor', '')
 " }}} Options "
 
 " Commands {{{ "
-command!       -nargs=? -complete=custom,cmake4vim#CompleteTarget    CMake                call cmake4vim#GenerateCMake(<f-args>)
-command!       -nargs=? -complete=custom,cmake4vim#CompleteTarget    CMakeResetAndReload  call cmake4vim#ResetAndReloadCMake(<f-args>)
-command!       -nargs=? -complete=custom,cmake4vim#CompleteTarget    CMakeBuild           call cmake4vim#CMakeBuild(<f-args>)
-command!       -nargs=1 -complete=custom,cmake4vim#CompleteTarget    CMakeSelectTarget    call cmake4vim#SelectTarget(<f-args>)
-command!       -nargs=1 -complete=custom,cmake4vim#CompleteBuildType CMakeSelectBuildType call cmake4vim#SelectBuildType(<f-args>)
-command!       -nargs=1 -complete=custom,cmake4vim#CompleteKit       CMakeSelectKit       call cmake4vim#SelectKit(<f-args>)
+command!       -nargs=? -complete=custom,cmake4vim#CompleteTarget       CMake                   call cmake4vim#GenerateCMake(<f-args>)
+command!       -nargs=? -complete=custom,cmake4vim#CompleteTarget       CMakeResetAndReload     call cmake4vim#ResetAndReloadCMake(<f-args>)
+command!       -nargs=? -complete=custom,cmake4vim#CompleteTarget       CMakeBuild              call cmake4vim#CMakeBuild(<f-args>)
+command!       -nargs=1 -complete=custom,cmake4vim#CompleteTarget       CMakeSelectTarget       call cmake4vim#SelectTarget(<f-args>)
+command!       -nargs=1 -complete=custom,cmake4vim#CompleteBuildType    CMakeSelectBuildType    call cmake4vim#SelectBuildType(<f-args>)
+command!       -nargs=1 -complete=custom,cmake4vim#CompleteKit          CMakeSelectKit          call cmake4vim#SelectKit(<f-args>)
+command!       -nargs=? -complete=custom,cmake4vim#CompleteCCMakeModes  CCMake                  call cmake4vim#CCMake(<f-args>)
```

Comment:
Is autocomplete for 1 letter actually necessary?

Reply:
Maybe you are right, I didn't have a time to finalize this PR.

Reply:
I keep the autocomplete, but I have changed names of Window modes.